<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow &gt; QWidget {
    background-image: url(:/new/img/bg.png);
}

#statusbar {
	background: black;
	text-align: center;
	color: white;
}

#menuBar {
	background: black;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidgetQueue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>321</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
	background-color: rgba(255, 255, 255, .2);
	border: 2px solid black;
	font-weight: bold;
	color: white;
	font-family: Arial;
	font-size: 20px;
}

QListWidget::item {
	border-bottom: 1px solid black;
	background-color: rgba(170, 0, 255, 0.7);
	height: 30px;
}

QListWidget::item::hover {
	background-color: black;
	color: white;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>621</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
color: white;
background-color: rgba(170, 0, 255, 0.5);
padding: 5px;
border: 3px solid black;
border-radius: 10px;
font: &quot;Arial&quot;;
font: &quot;Arial Rounded MT Bold&quot;;
font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>Olivia Rodrigo's Concert Ticketing System</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #592693;
color: white;
border: 3px solid black;
border-radius: 5px;
font-weight: bold;
font-size: 24px;</string>
    </property>
    <property name="text">
     <string>Queue Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNextInLine">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #592693;
font-weight: bold;
color: white;</string>
    </property>
    <property name="title">
     <string>                      Purchase a Ticket</string>
    </property>
    <widget class="QLabel" name="nameWarning">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Enter name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTicketAdded">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-size: 15px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="enQueue">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #592693;
	color: white;
	font-weight: bold;
	border: 2px solid black;
	border-radius: 5px;
}

QPushButton::hover {
	background-color: rgba(255, 255, 255, 0.6);
}

QPushButton::clicked {
	background-color: rgba(0, 0, 0, 0.6);
}</string>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: #592693;
	color: white;
	border: 2px solid black;
	border-radius: 5px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberTimer">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000;
border-radius: 10px;
border: 2px solid white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTicketReceived">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSearch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #592693;
color: white;
border: 2px solid black;
border-radius: 5px;
padding: 5px;
font: 14pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelQueueSize">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #592693;
color: white;
border: 3px solid black;
border-radius: 5px;
font-weight: bold;
font-size: 24px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuh">
    <property name="title">
     <string>Queue time</string>
    </property>
    <addaction name="action60s"/>
    <addaction name="action30s"/>
    <addaction name="action10s"/>
   </widget>
   <addaction name="menuh"/>
  </widget>
  <action name="action60s">
   <property name="text">
    <string>60s</string>
   </property>
  </action>
  <action name="action30s">
   <property name="text">
    <string>30s</string>
   </property>
  </action>
  <action name="action10s">
   <property name="text">
    <string>10s</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
